docs/configuration/single-process-config-blocks.yaml

```diff
@@ -28,7 +28,7 @@ ingester_client:
     use_gzip_compression: true
 
 ingester:
-  # Disable blocks transfers on ingesters shutdown or rollout.
+  # Blocks storage doesn't support blocks transfer between ingesters on shutdown or rollout.
```

Comment:
Remove this option completely? (in all YAML files using blocks)

Reply:
Better.